docs/snippets/aws-acm-external-secret.yaml

```
@@ -0,0 +1,23 @@
+
```

YAMLlint (1.38.0)

[warning] too many blank lines (1 > 0)

(empty-lines)